Can't paste a function

If I run `ts-node` from within my project, and I paste the line `function makeSignObject(signs: Array<string>): Array<object> {` into the repl, I expected that it would, like node, let me finish writing the function definition on a new line. Instead, it reports `[eval].ts(2,1): error TS1005: '}' expected.
[eval].ts(1,48): error TS2355: A function whose declared type is neither 'void' nor 'any' must return a value.` 

How do you paste functions into ts-node and use it as an actual repl?

--- a/src/bin.ts
+++ b/src/bin.ts
@@ -387,7 +387,8 @@ const RECOVERY_CODES: Set<number> = new Set([
   1109, // "Expression expected."
   1126, // "Unexpected end of text."
   1160, // "Unterminated template literal."
-  1161 // "Unterminated regular expression literal."
+  1161, // "Unterminated regular expression literal."
+  2355 // "A function whose declared type is neither 'void' nor 'any' must return a value."
 ])
 
 /**

